Apps/Inc/Tasks.h

```diff
@@ -134,36 +128,60 @@ typedef enum{
     OS_MAIN_LOC = 0x200,
     OS_CANDRIVER_LOC = 0x400,
     OS_MOTOR_CONNECTION_LOC = 0x800,
-    OS_DISPLAY_LOC = 0x1000
+    OS_DISPLAY_LOC = 0x1000,
+    OS_TASKS_LOC = 0x2000,
 } os_error_loc_t;
 
-/**
- * Fault Enum
- * 
- * Different fault states that need to be handled by the FaultState task
- */
-typedef enum{
-    FAULT_NONE = 0x00,      // No fault
-    FAULT_OS = 0x01,         // for OS faults
-    FAULT_UNREACH = 0x02,    // for unreachable conditions
-    FAULT_TRITIUM = 0x04,      // for errors sent from the tritium
-    FAULT_READBPS = 0x08,    // for unsuccessfully reading from BPS CAN
-    FAULT_DISPLAY = 0x10,    // for display faults
-    FAULT_BPS = 0x20,       // for if BPS trips
-} fault_bitmap_t;
-
 /**
  * Error variables
  */
-extern fault_bitmap_t FaultBitmap;
 extern os_error_loc_t OSErrLocBitmap;
 
+// Store error codes that are set in task error assertion functions
+extern error_code_t Error_ReadTritium; 
+extern error_code_t Error_ReadCarCAN;
+extern error_code_t Error_UpdateDisplay;
+
+/**
+ * Error-handling option enums
+*/
+
+// Scheduler lock parameter option for asserting a task error
+typedef enum {
+    OPT_LOCK_SCHED = false,
+    OPT_NO_LOCK_SCHED = true
+} error_lock_sched_opt_t;
```

Comment:
This enum name is kind of weird. Maybe `error_scheduler_lock_opt_t` is better but it could also just be me being dumb

Reply:
I'll change it- `error_scheduler_lock_opt_t` gives a better idea of what it's used for.